### DISCUSSION TITLE
Wifi streaming stop when the drone take off

### DISCUSSION THREAD

**Discussion by HeleMartin** [bitcraze/discussions#116] [Q&A] [answered]
Hello,
I'm trying to use the AIdeck to make my own wifi image streaming. 
The image streaming works when the battery of the drone is connected to the PC and the propellers start. But when the drone uses its own battery, the image streaming stops when the propellers start.
Do you know this issue ?

**Comment by tobbeanton** [comment, marked as answer]
When propellers start, do you mean take off? The ESP32 seems a bit sensitive to low voltage levels and when the motors spin up quickly from standing still they require a lot of energy resulting in a voltage dip. This can be mitigated by first spinning up the propellers a bit before the take off. I A battery that is bad or a connector that is oxidized would also add to the problem. Try cleaning the connector.

**Reply by HeleMartin** [reply]
Hello,
Yes, I mean when the drone take off. I have checked the connector, it is ok. How can I spin upt the propellers a bit before the take off ? I'm using an app layer and I have to give an height.

**Reply by HeleMartin** [reply]
Hello,
I have succeeded in taking off the drone with wifi image streaming.
I have taken off the drone at 0.0 meters during 1 second and then I have launched the take off command at the hope height.

**Reply by tobbeanton** [reply]
I'm happy it worked out!

I think if using the high level commander the take off can be made smother with the takeoff_with_velocity(...) function.

I'm adding [this](https://github.com/bitcraze/crazyflie-firmware/issues/695) reference issue here as it is also relevant for the lighthouse deck.
